### ISSUE TITLE
Abandoned

### ISSUE THREAD

**Issue by chiquitinxx** [chiquitinxx/grooscript#68]
Hey!

No activity, no one maintaining, out of date in libraries and dependencies, lack of interest. So, I'll stop paying domain soon, close twitter account and declare the project as "dead". Will stay here in Github, is free :) If someone is interested to continue, ping me.

Thank you all, many interest at the beginning, I learned a lot and has been amazing journey.

**Comment by cdrchops** [comment]
let me know what you need to do.  I've been working on upgrading the libraries locally and added some new methods for the abiliity to add Javascript.  I'd be happy to take it over if that's something you're interested in.

**Comment by chiquitinxx** [comment]
Hello!

Very happy if you continue as mantainer of the project. It seem you have a grooscript repository in your user and I can't transfer the ownership of the github repository. Maybe better if I just add a link to your repository and you continue there?

Also let me know what you want to do with the www domain, and the maven credentials to push to maven central.

Thank you!

**Comment by cdrchops** [comment]
Yes I branched the project so I could add some work to it.  Sure.  Let’s
just make a link to my project.  We can transfer ownership of the domain I
think.   And I can take over the hosting either where it is or on my
server.

Should I get new credentials to mavencentral?

**Comment by cdrchops** [comment]
Also, I branched the Grooscript plugin project as well.  I may still have some questions in the future if it's ok to ask you

**Comment by chiquitinxx** [comment]
Sure, I'll update README's to point your clones, and will start to pass things to you. Please send me your email to grooscript@gmail.com for better comunication. It's summer time, take it easy :) Thanks for volunteer
